test/command/5849-prefix.md

```diff
@@ -0,0 +1,8 @@
+```
+% pandoc -t latex --biblatex
+[e.g. @a1;@a2;@a3; but also @b1;@b2;@b3]
+^D
+\autocites[e.g.~][]{a1}{a2}{a3}[but also][]{b1}{b2}{b3}
+```
```

Comment:
Is this an adequate test for this feature?
Because I get exactly the same output with current pandoc on this input.

Reply:
This revised test output looks right.
Maybe it would be worth testing suffixes as well as prefixes?  Something like
```
[e.g. @a1, ch. 3 and elsewhere; @a2; @a3; but also @a4; @a5]
```

Reply:
what would be the output for that test? I've writtern some code, but I've made the output like this:
`\autocites[e.g.~][ch. 3 and elsewhere]{@a1}[e.g.~][]{@a2}{@a3}[but also][]{@a4}{@a5}` Is this right?